index.html

```diff
@@ -22,7 +22,7 @@
         <h1 class="lor">Lorem Ipsum Dolor</h1>

```

Comment:
pls give the class more meaningful name. PS: The class's name should describe what class does. For example **main-heading** or something like that. Pls go through all classes and check one more time that classes names describe what they do

Reply:
Done